src/components/include/sl.h

```diff
@@ -526,6 +528,6 @@ sl_cs_exit_switchto(struct sl_thd *to)
  * sl_sched_loop(); <- loop here
  */
 void sl_init(microsec_t period);
-void sl_sched_loop(void);
+void sl_sched_loop(int non_block);
```

Comment:
Shouldn't this has an `enum` type or something?  Or is it flags?  The use of this is very unclear to me.